mappings/net/minecraft/village/TraderOfferList.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_86 net/minecraft/village/TraderOfferList
 	METHOD method_148 toTag ()Lnet/minecraft/class_1405;
+	METHOD method_149 (Lnet/minecraft/class_1405;)V
+		ARG 1 compundTag
```

Comment:
Typo in compundTag -> compoundTag
Pls fix before merge otherwise there will be a merge conflict

Reply:
alright I will get to it